### ISSUE TITLE
Triggered by viral infection

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6235]
**Preferred term label:**
Triggered by viral infection

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://omim.org/entry/613759
https://pubmed.ncbi.nlm.nih.gov/21109225/

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
